### DISCUSSION TITLE
Discord Invite and Questions

### DISCUSSION THREAD

**Discussion by headsetsniper** [Chizaruu/com.tsk.ide.vscode#57] [General]
Hi,
The discord invite on the button seems to be invalid / expired. 
Can someone fix that / send me an invite, I am very interested in this extension, but beyond the initial setup, I dont really seeknow how to use this extension. On the vscode side, Tests are not showing up in the test explorer and dotnet build / test also gives errors, but I´m pretty sure, Im just not getting the whole picture. Windows, Unity 2022.3.0 btw.

**Comment by Chizaruu** [comment]
The link should be fixed now.

Also, it's a Unity package, not an extension :o

**Reply by headsetsniper** [reply]
Eh yeah, dont know where that came from XD plugin of course. But the Discord Link still doesnt work : 
![image](https://github.com/Chizaruu/com.tsk.ide.vscode/assets/3028377/14615d34-85ad-43c2-a8a1-323e0d888b4f)
Tested on Edge and Firefox

**Reply by Chizaruu** [reply]
https://discord.com/invite/tZmsjTFEBx ?
